### ISSUE TITLE
Possible to pass multiple files to `convert_file`

### ISSUE THREAD

**Issue by simonv3** [NicklasTegner/pypandoc#248]
At the command line this is okay:

```
pandoc _posts/*.md \
  --from markdown \
  -o books/cookbook.epub
```

but it looks like if I do the same with `pypandoc.convert_file('_posts/*.md', 'epub', outputfile="books/test.epub")` I get this error:

```
Traceback (most recent call last):
  File "book_creator.py", line 26, in <module>
    pypandoc.convert_file('_posts/2022-01-12-vegan-shakshuka.md _posts/2022-01-12-vegan-dahl.md', 'epub', outputfile="books/test.epub")
  File "/home/simon/src/symbiosis-cookbook/.venv/lib/python3.8/site-packages/pypandoc/__init__.py", line 148, in convert_file
    raise RuntimeError("source_file is not a valid path")
RuntimeError: source_file is not a valid path
```

So I tried passing multiple files to the converter, listed individually:

```
pypandoc.convert_file('_posts/2022-01-12-vegan-shakshuka.md _posts/2022-01-12-vegan-dahl.md', 'epub', outputfile="books/test.epub")
```

which gives the same error.

Is there a way to pass multiple files to pandoc like on the command line? (I'm basically just using python to sort the markdown files by frontmatter category).

**Comment by NicklasTegner** [comment]
I'll look into this. This would indeed be nice to have, but it would take some restructuring to accomplish.

**Comment by simonv3** [comment]
For me there's no rush, I'm just sending the command straight to `subprocess.check_call` which is fine cause I'm the only one using the script atm.

**Comment by NicklasTegner** [comment]
@simonv3  let me know if what I've implemented in #259  works for you.
Any feedback / suggestions etc
